Wrong shot number:
[3, 15, 9]

Correct shot number:
[13]

Raw json:

{'hasError': True, 'errParameter': ['fs.s3a.s3guard.ddb.table.create'], 'reason': ["The value of the property 'fs.s3a.s3guard.ddb.table.create' should be a boolean value."]}

{'hasError': True, 'errParameter': ['hadoop.caller.context.enabled'], 'reason': ["The value of the property 'hadoop.caller.context.enabled' is not within the accepted value {true,false}."]}

{'hasError': True, 'errParameter': ['hadoop.caller.context.enabled'], 'reason': ["The property 'hadoop.caller.context.enabled' has the value 'true' which is not within the accepted value {true,false}."]}

Final result:

There are 1 misconfiguration parameters in the input: hadoop.caller.context.enabled
[Ciri] Reason for hadoop.caller.context.enabled: The value of the property 'hadoop.caller.context.enabled' is not within the accepted value {true,false}.
